apps/prairielearn/elements/pl-symbolic-input/pl-symbolic-input.py

```diff
@@ -49,18 +47,39 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
         "blank-value",
         "placeholder",
         "custom-functions",
+        "show-score",
     ]
     pl.check_attribs(element, required_attribs, optional_attribs)
     name = pl.get_string_attrib(element, "answers-name")
     pl.check_answers_names(data, name)
 
-    correct_answer = pl.get_string_attrib(
-        element, "correct-answer", CORRECT_ANSWER_DEFAULT
-    )
-    if correct_answer is not None:
+    if pl.has_attrib(element, "correct-answer"):
```

Comment:
A tiny bit orthogonal to the purpose of this PR, but I included this here because the code could essentially be copied directly from the bigo element. All this does is parse the correct answer during prepare and display an error message if the parsing failed.